include/pinproc.h

```diff
@@ -138,6 +138,9 @@ PINPROC_API PRResult PRFlushWriteData(PRHandle handle);
 /** Write data out to the P-ROC immediately (does not require a call to PRFlushWriteData). */
 PINPROC_API PRResult PRWriteData(PRHandle handle, uint32_t moduleSelect, uint32_t startingAddr, int32_t numWriteWords, uint32_t * writeBuffer);
 
+/** Write data buffered to P-ROC (does require a call to PRFlushWriteData). */
+PINPROC_API PRResult PRWriteDataUnbuffered(PRHandle handle, uint32_t moduleSelect, uint32_t startingAddr, int32_t numWriteWords, uint32_t * writeBuffer);
+
```

Comment:
This is a new API from @jabdoa2 and @Compy.

Reply:
This is used for fading in MPF (via pypinproc MPF fork with GIL fixes)